### PR TITLE
feat(lpms-cli): added `lpms-cli` package

### PR DESCRIPTION
**Pull request by kostysh**
`lpms-cli` package contains a CLI utility dedicated to interact with `lpms-server` RESTful API
This version of the utility supports only the following commands:
- `version`: prints version of the package
- `mnemonic`: generates a random 24-word mnemonic

Package not published to NPM registry yet

### CODE REVIEW

**Comment by mfw78** [comment]
With respect to the package, should we not in include `dist/` in the `.gitignore` so that we can keep the repository clean?

File: .github/workflows/dispatch.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        run: yarn exec --scope @windingtree/${{ github.event.client_payload.packageName }} yarn build
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          registry-url: "https://registry.npmjs.org"
+          scope: "@windingtree"
+      - name: Publish to npmjs.org
+        run: yarn exec --scope @windingtree/${{ github.event.client_payload.packageName }} npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
```
Comment (mfw78) [inline .github/workflows/dispatch.yml:27]: Please change `NPM_TOKEN` to align with repository-wide secret for `npmjs` publishing.

**Review by mfw78 — CHANGES_REQUESTED** [review]
Changes required for the `NPM_AUTH_TOKEN`. Otherwise all good!

File: .github/workflows/dispatch.yml (hunk 1 above)
Comment (kostysh) [reply .github/workflows/dispatch.yml]: `NPM_TOKEN` IS CHANGED TO `NPM_AUTH_TOKEN`

**Review by mfw78 — COMMENTED** [review]
LGTM!

**Review by mfw78 — COMMENTED** [review]
LGTM!

**Pull request merged**